benchmarks/bc_bench.cu

```diff
@@ -73,19 +73,18 @@ void bc_bench(nvbench::state& state) {
 
   // --
   // Build graph + metadata
-  csr_t csr;
   io::matrix_market_t<vertex_t, edge_t, weight_t> mm;
-  csr.from_coo(mm.load(filename));
-
-  auto G = graph::build::from_csr<memory_space_t::device,
-                                  graph::view_t::csr>(
-      csr.number_of_rows,               // rows
-      csr.number_of_columns,            // columns
-      csr.number_of_nonzeros,           // nonzeros
-      csr.row_offsets.data().get(),     // row_offsets
-      csr.column_indices.data().get(),  // column_indices
-      csr.nonzero_values.data().get()   // values
-  );
+  gunrock::io::loader_struct<vertex_t, edge_t, weight_t> loader;
+  loader = mm.load(filename);
+  
+  format::csr_t<memory_space_t::device, vertex_t, edge_t, weight_t> csr;
+  csr.from_coo(loader.coo);
+
+  // --
+  // Build graph
+
+  auto G =
+      graph::build::build<memory_space_t::device>(loader.properties, csr);
```

Comment:
I will probably try to improve this bit. The `.load()` can return two things:
```cpp
auto [properties, coo] = mm.load(filename);
```

https://stackoverflow.com/questions/321068/returning-multiple-values-from-a-c-function

And the graph could then be built:
```cpp
format::csr_t<memory_space_t::device, vertex_t, edge_t, weight_t> csr;
csr.from_coo(coo);
auto G = graph::build::build<memory_space_t::device>(properties, csr);
```

Reply:
This is updated.